demisto_sdk/commands/common/update_id_set.py

```diff
@@ -142,7 +142,17 @@ def get_commmands_from_playbook(data_dict):
     return command_to_integration
 
 
-def get_integration_data(file_path):
+def get_integration_api_modules(file_path, data_dictionary, is_unified_integration):
+    unifier = Unifier(os.path.dirname(file_path))
+    if is_unified_integration:
+        integration_script_code = data_dictionary.get('script', {}).get('script', '')
+    else:
+        _, integration_script_code = unifier.get_script_package_data()
```

Comment:
let's update the name of get_script_package_data it no longer works only for scripts